lib/redux/actions/index.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2015, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import * as asyncActions from './async';
+import * as syncActions from './sync';
+
+export default {
+  asyncActions: asyncActions,
```

Comment:
I do still find this kind of weird, with the repetition of the word 'action' e.g.

```
import action from './lib/redux/actions'

actions.asyncActions.doSomething()
```

If you do just import the async actions at one point you can `import { async as asyncActions }` from './lib/redux/actions'`

```
import { async as asyncActions }` from './lib/redux/actions`

asyncActions.doSomething()
```

Again preference here, no big deal either way.